managed-ledger/src/main/java/org/apache/bookkeeper/mledger/impl/ManagedCursorImpl.java

```diff
@@ -1823,8 +1825,10 @@ void internalFlushPendingMarkDeletes() {
 
     void createNewMetadataLedger(final VoidCallback callback) {
         ledger.mbean.startCursorLedgerCreateOp();
+        final long now = System.currentTimeMillis();
```

Comment:
Can we measure the latency at the lower level call, in places like zkutils instead of doing it at higher level?